src/js/components/DescriptionList.js

```diff
@@ -1,6 +1,12 @@
+import PureRenderMixin from 'react-addons-pure-render-mixin';
 import React from 'react';
 
 class DescriptionList extends React.Component {
+  constructor() {
+    super(...arguments);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate;
```

Comment:
Nope, just looked at that resource and figured we should follow it.
Looking into it a bit more, you are right. It shouldn't matter: https://github.com/facebook/react/blob/master/src/addons/ReactComponentWithPureRenderMixin.js

[See Babel example here](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Ces2016%2Ces2017%2Creact%2Cstage-2&experimental=false&loose=false&spec=true&playground=false&code=let%20mixin%20%3D%20function%20()%20%7B%0A%20%20console.log(this.constructor)%3B%0A%7D%0A%0Aclass%20Hello%20%7B%0A%20%20constructor(prop)%20%7B%0A%20%20%20%20this.mixin%20%3D%20mixin%3B%0A%20%20%7D%0A%7D%0A%0Aclass%20There%20%7B%0A%20%20constructor(prop)%20%7B%0A%20%20%20%20this.mixin%20%3D%20mixin%3B%0A%20%20%7D%0A%7D%0A%0Alet%20hello%20%3D%20new%20Hello()%3B%0Ahello.mixin()%3B%0Alet%20there%20%3D%20new%20There()%3B%0Athere.mixin()%3B)